src/atoms/feedCardItem/types.ts

```diff
@@ -0,0 +1,61 @@
+export interface TransactionData {
+  hash: string;
+  timestamp: number;
+  block: number;
+  wallet: string;
+  from: string;
+  to: string;
+  tx_hash: string;
+  tx_type: string;
+  dex: '0x'
```

Comment:
Ah rather than typing all the keys out you can just do `keyof typeof <obj>` and it will be the exact same type as you have here right now